environments/prod/mailrelay-platform-hmcts-net.yml

```diff
@@ -34,7 +34,4 @@ A:
 cname:
   - name: "dev-in"
     ttl: 60
-    record: "ss-dev-mailrelay.trafficmanager.net"
-  - name: "prod-in"
-    ttl: 60
-    record: "ss-prod-mailrelay.trafficmanager.net"
```

Comment:
why is this getting removed here?

Reply:
I asked him to remove it, clashing with the A record, i'll circle back to mailrelay next spring. The branch fails on merge if this is in

Reply:
should be removed in a separate PR though and not this one

Reply:
fair point, i am just looking for a clean build on merge this time for this repo. 👌🏾